bindings/nodejs/types/wallet/signedTransactionEssence.ts

```diff
@@ -1,8 +1,8 @@
 import type { ITransactionPayload } from '@iota/types';
-import type { InputSigningData } from './preparedTransactionData';
+import { IInputSigningData } from '../client';
 
 /** The signed transaction with inputs data */
 export interface SignedTransactionEssence {
     transactionPayload: ITransactionPayload;
-    inputsData: InputSigningData;
+    inputsData: IInputSigningData;
```

Comment:
Can you explain this to a TS rookie?

Reply:
Interfaces should have the I prefix. importing a type or not makes no difference so i removed it.

Client exports the preparedTransactionData interfaces atm, the other preparedTransactionData is the wrapper class